### PR TITLE
add cluster-admin role

### PR DESCRIPTION
**Pull request by kahootali**

### CODE REVIEW

**Comment by stakater-user** [comment]
@kahootali Chart: prometheus-operator Dry Run successful

File: prometheus-operator/templates/rbac.yaml [hunk 1]
```diff
@@ -80,3 +80,27 @@ subjects:
 - kind: ServiceAccount
   name: {{ template "name" . }}
   namespace: {{ .Release.Namespace }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: cluster-admin
```
Comment (hazim1093) [inline prometheus-operator/templates/rbac.yaml]: Won't this fail for @rasheedamir as they already have the role `cluster-admin`? @kahootali

Comment (rasheedamir) [reply prometheus-operator/templates/rbac.yaml]: won't it skip if its there already? it has to be there as we will be deploying to different clusters

Comment (kahootali) [reply prometheus-operator/templates/rbac.yaml]: Most probably, it will give an error that a service account with this name already exists.

Comment (rasheedamir) [reply prometheus-operator/templates/rbac.yaml]: lets give name like po-cluster-admin

prometheus operator cluster admin

**Pull request merged**